stmhal/boards/stm32f429.ld

```diff
@@ -0,0 +1,130 @@
+/*
+    GNU linker script for STM32F429i-Discovery kit with external 8MByte SDRam
+*/
+
+/* Specify the memory areas */
+MEMORY
+{                                                   
+    FLASH (rx)      : ORIGIN = 0x08000000, LENGTH = 0x0200000 /* entire flash, 2048 KiB */
+    FLASH_ISR (rx)  : ORIGIN = 0x08000000, LENGTH = 0x0004000 /* sector 0, 16 KiB */
+    FLASH_TEXT (rx) : ORIGIN = 0x08020000, LENGTH = 0x0088000 /* sectors 5,6,7,8, 4*128KiB = 512 KiB (could increase it more) */
+    RAM (xrw)       : ORIGIN = 0x20000000, LENGTH = 0x0030000 /* 192 KiB */
```

Comment:
The stm32f429 has 256 kByte RAM but only 192 are at 0x20000000 the remaining 64kByte is at 0x10000000 (CCM).